code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -50,6 +53,9 @@
 	/// Whether this implant has already triggered on death or not, to prevent it firing multiple times.
 	var/has_triggered_on_death = FALSE
 
+	///the implant_fluff datum attached to this implant, purely cosmetic "lore" information
+	var/datum/implant_fluff/implant_data = new()
```

Comment:
```dm
	var/datum/implant_fluff/implant_data = /datum/implant_fluff
```
 instead of `=new()` and instantiate it in the Initialize(). i.e.
```dm
/Initialize(mapload)
  . = ..()
  if(ispath(implant_data)) implant_data = new implant_data
```

Reply:
sounds good

Reply:
added